src/Compiler/FSComp.txt

```diff
@@ -1672,4 +1672,5 @@ featureEscapeBracesInFormattableString,"Escapes curly braces before calling Form
 3555,chkInstanceLetBindingOnStaticClasses,"If a type uses both [<Sealed>] and [<AbstractClass>] attributes, it means it is static. Instance let bindings are not allowed."
 3556,chkImplementingInterfacesOnStaticClasses,"If a type uses both [<Sealed>] and [<AbstractClass>] attributes, it means it is static. Implementing interfaces is not allowed."
 3557,chkAbstractMembersDeclarationsOnStaticClasses,"If a type uses both [<Sealed>] and [<AbstractClass>] attributes, it means it is static. Abstract member declarations are not allowed."
-3558,chkExplicitFieldsDeclarationsOnStaticClasses,"If a type uses both [<Sealed>] and [<AbstractClass>] attributes, it means it is static. Explicit field declarations are not allowed."
\ No newline at end of file
+3558,chkExplicitFieldsDeclarationsOnStaticClasses,"If a type uses both [<Sealed>] and [<AbstractClass>] attributes, it means it is static. Explicit field declarations are not allowed."
+3559,typrelNeverRefinedAwayFromTop,"A type has been implicitly inferred as `obj`, which may be unintended. Consider adding explicit type annotations, disabling the warning, or suppressing it locally with `#nowarn \"3559\"`."
```

Comment:
Don't use backticks in the messages. See, for instance, line 1658 above.

```suggestion
3559,typrelNeverRefinedAwayFromTop,"A type has been implicitly inferred as 'obj', which may be unintended. Consider adding explicit type annotations, disabling the warning, or suppressing it locally with '#nowarn \"3559\"'."
```

Reply:
Actually, I think it is better to be consistent with the other wording used above. Consistency across the board is better. 

I mean:

```suggestion
3559,typrelNeverRefinedAwayFromTop,"A type has been implicitly inferred as `obj`, which may be unintended. Consider adding explicit type annotations. You can disable this warning by using '#nowarn \"3535\"' or '--nowarn:3535'."
```